Source/ACE/Network/GameMessages/Messages/GameMessageFellowshipFullUpdate.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ACE.Common.Extensions;
+using ACE.Entity.Actions;
+using ACE.Entity;
+
+namespace ACE.Network.GameMessages.Messages
+{
+    public class GameMessageFellowshipFullUpdate : GameMessage
+    {
+        public GameMessageFellowshipFullUpdate(Session session)
+            : base(GameMessageOpcode.GameEvent, GameMessageGroup.Group09)
+        {
+            Fellowship fellowship = session.Player.Fellowship;
+            ActionChain chain = new ActionChain();
+            chain.AddAction(session.Player, () =>
+            {
+                Writer.Write(session.Player.Guid.Full);
+                Writer.Write(session.GameEventSequence++);
+                Writer.Write((uint)GameEvent.GameEventType.FellowshipFullUpdate);
+
+                // the current number of fellowship members
+                Writer.Write((UInt16)fellowship.FellowshipMembers.Count);
+
+                // ????
+                Writer.Write((byte)0x10);
+                Writer.Write((byte)0x00);
+
+                // --- FellowInfo ---
+                
+                foreach (Player fellow in fellowship.FellowshipMembers)
+                {
+                    if (fellow.Guid.Full != session.Player.Guid.Full)
+                    {
+                        chain.AddAction(fellow, () => { WriteFellow(fellow); });
+                    } else
+                    {
+                        WriteFellow(fellow);
+                    }
+                }
+
+                Writer.WriteString16L(fellowship.FellowshipName);
+
+                // guid of fellowship leader
+                Writer.Write(fellowship.FellowshipLeaderGuid);
+
+                // todo: xp share?
+                Writer.Write((byte)0x01);
+                Writer.Write((byte)0x00);
+                Writer.Write((byte)0x00);
+                Writer.Write((byte)0x00);
+
+                // todo: open fellow?
+                Writer.Write((byte)0x01);
+                Writer.Write((byte)0x00);
+                Writer.Write((byte)0x00);
+                Writer.Write((byte)0x00);
+
+                // todo: fellows departed?
+                Writer.Write((byte)0x00);
+                Writer.Write((byte)0x00);
+                Writer.Write((byte)0x00);
+                Writer.Write((byte)0x00);
+
+                Writer.Write(0u);
+
```

Comment:
This is locked. (uint)1 or (uint)0. (You'll notice most of the items that are technically bools, are still typically still 4 byte uints.)